packages/sites/test/pages/convert-page-to-template.test.ts

```diff
@@ -23,22 +23,22 @@ describe("convertPageToTemplate", () => {
         folderId: "delete",
         slug: "delete",
         testIdReplacement: "page-id",
-        layout: testLayout
-      }
-    }
-  } as unknown) as IModel;
+        layout: testLayout,
+      },
+    },
+  } as unknown as IModel;
 
   const ro = {
-    isPortal: false
+    isPortal: false,
   } as IHubRequestOptions;
 
   const assets: ITemplateAsset[] = [
     {
       mimeType: "image/png",
       name: "some-name",
       url: "some-url",
-      type: "some-type"
-    }
+      type: "some-type",
+    },
```

Comment:
you should add a `draft-1234123.json` into this and then ensure it's removed, thus verifying that the `.filter` is working as expected